### PR TITLE
Fix router

### PR DESCRIPTION
**Pull request by stasbar**

### CODE REVIEW

File: src/models/batteries.ts [hunk 1]
```diff
@@ -53,107 +9,25 @@ export interface BatteriesModelState {
...
-        ...(state as BatteriesModelState),
-        showNewAffiliateModal: true,
-      };
-    },
-    toggleEditBattery(state = { batteries: [] }): BatteriesModelState {
-      return {
-        ...(state as BatteriesModelState),
-        editBattery: !state.editBattery,
-      };
-    },
```
Comment (stasbar) [inline src/models/batteries.ts:113]: bring back this function

File: src/components/BatteryPreviewDrawer/index.tsx [hunk 2]
```diff
@@ -50,7 +53,7 @@ const BatteryPreviewDrawer: React.FC<BatteryPreviewDrawerProps> = (
       {content}
     </div>
   );
-  const toggleEditBattery = () => dispatchToggleEditBattery(dispatch);
+  const toggleEditBattery = () => setEditBattery(selectedBattery);
```
Comment (stasbar) [inline src/components/BatteryPreviewDrawer/index.tsx:56]: it is `toggleEditBattery` not `setEditBattery `

File: src/pages/liquid/LiquidBlender.tsx [hunk 3]
```diff
@@ -255,24 +255,18 @@ const LiquidBlender: React.FC<LiquidBlenderProps> = ({
...
-                      price: `${result.price.toFixed(2)}${formatMessage({
-                        id: 'app.currency',
-                        defaultMessage: '$',
-                      })}`,
-                      weight: `${result.weight.toFixed(3)} ${formatMessage({
-                        id: 'misc.units.gram',
-                        defaultMessage: 'g',
-                      })}`,
+                      price: `${result.price.toFixed(2)}${(<Typography>$</Typography>)}`,
+                      weight: `${result.weight.toFixed(3)} ${(<Typography>g</Typography>)}`,
```
Comment (stasbar) [inline src/pages/liquid/LiquidBlender.tsx:269]: Bring back `formatMessage`

File: src/pages/mixer/inputElements.tsx [hunk 4]
```diff
@@ -56,22 +56,19 @@ export default function InputElements(props: any) {
...
               props.onValueChange({
                 ...props.mixData,
                 strength,
               });
             }}
-            placeholder={formatMessage({
-              id: 'mixer.strength',
-              defaultMessage: 'Strength [mg/ml]',
-            })}
+            placeholder="Strength"
```
Comment (stasbar) [inline src/pages/mixer/inputElements.tsx:65]: bring back `formatMessage`

File: src/pages/mixer/inputElements.tsx [hunk 5]
```diff
@@ -92,16 +89,13 @@ export default function InputElements(props: any) {
...
               props.onValueChange({
                 ...props.mixData,
                 flavorPercentage: newValue,
               });
             }}
-            placeholder={formatMessage({
-              id: 'mixer.flavorPercentage',
-              defaultMessage: 'Flavor [%]',
-            })}
+            placeholder="Flavor [%]"
```
Comment (stasbar) [inline src/pages/mixer/inputElements.tsx:98]: bring back `formatMessage`

**Comment by gbaranski** [comment]
Commenting works with links and photos, with posts doesn't work bcs of permission denied, i think issue may be becuase posts are links for some reason, but first to fix is uploading.

**Comment by gbaranski** [comment]
```
sla = log --graph --abbrev-commit --decorate --format=format:'%C(bold blue)%h%C(reset) - %C(bold green)(%ar)%C(reset) %C(white)%s%C(reset) %C(dim white)- %an%C(reset)%C(bold yellow)%d%C(reset)' --all
```

File: src/models/coil.ts [hunk 6]
```diff
@@ -1,264 +1,119 @@
...
-    [SET_PROPERTIES]: Reducer<CoilModelState>;
-    [SET_TYPE]: Reducer<CoilModelState>;
-    [SET_WIRE]: Reducer<CoilModelState>;
-    [DELETE_WIRE]: Reducer<CoilModelState>;
-    [ADD_WIRE]: Reducer<CoilModelState>;
+export default () => {
+  const [coil, setCoil] = useState<Coil>(wireGenerator.normalCoil());
+  const [voltage, setVoltage] = useState<number>(3.7);
+  const [properties, setProperties] = useState<Properties>();
+  const [pathsState, setPathsState] = useState<any>({
```
Comment (stasbar) [inline src/models/coil.ts:17]: what is that ? It wasn't used anywhere before

**Comment by gbaranski** [comment]
@stasbar when you were sleeping i fixed adding posts and photos and also links, atm im trying to figure out this permissions problem.

**Pull request merged**